backend/app/tests/utils/utils.py

```
@@ -89,6 +91,49 @@ def get_project(session: Session, name: str | None = None) -> Project:
     return project
 
 
+from dotenv import load_dotenv, find_dotenv
```

Ruff (0.12.2)

Module level import not at top of file

(E402)